src/wp-includes/functions.wp-scripts.php

```diff
@@ -132,11 +132,11 @@ function wp_add_inline_script( $handle, $data, $position = 'after' ) {
  * @since WP-4.3.0 A return value was added.
  *
  * @param string           $handle    Name of the script. Should be unique.
- * @param string           $src       Full URL of the script, or path of the script relative to the WordPress root directory.
+ * @param string           $src       Full URL of the script, or path of the script relative to the ClassicPress root directory.
  * @param array            $deps      Optional. An array of registered script handles this script depends on. Default empty array.
  * @param string|bool|null $ver       Optional. String specifying script version number, if it has one, which is added to the URL
  *                                    as a query string for cache busting purposes. If version is set to false, a version
- *                                    number is automatically added equal to current installed WordPress version.
+ *                                    number is automatically added equal to current installed ClassicPress version.
```

Comment:
Not true:  this is still using the current installed WordPress version.  Let's either change the version or leave the text as `WordPress` until we do.